forest-node/ansible/roles/docker/tasks/main.yml

```diff
@@ -0,0 +1,120 @@
+---
+# tasks file for role/docker
+- name: Upgrade all apt packages
+  apt: upgrade=dist force_apt_get=yes
+
+- name: Create a login user
+  user:
+    name: "{{ user }}"
+    state: present
+
+- name: Add public key to authorized_keys
+  authorized_key:
+    user: "{{ user }}"
+    state: present
+    key: "{{ lookup('file', '~/id_rsa.pub') }}"
+
+- name: Deny root from logging in
+  ansible.builtin.lineinfile:
+    dest: /etc/ssh/sshd_config
+    regexp: '^(#)?PermitRootLogin \w*$'
+    line: 'PermitRootLogin no'
+    state: present
+
+- name: Restart SSHD Service 
+  become: true
+  shell: sudo service ssh restart 
+
+- name: Allow specific users to log in
+  ansible.builtin.lineinfile:
+    dest: /etc/ssh/sshd_config
+    regexp: '^AllowUsers'
+    line: 'AllowUsers {{ user }}'
+    state: present
+
+- name: Add {{ user }} to sudoers file
```

Comment:
It's practically a root user. Let's allow logging as' root' if we want to have a root user and leave the service user limited.

Reply:
Got it. I'd allow root user login and limit the service user.